homeworks/mariia.demchenko_demasha0908/GamesProject/index.html

```diff
@@ -4,28 +4,61 @@
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
-        <link rel="stylesheet" href="style.css">
-        <title>Document</title>
+        <title>Games project</title>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="css/style.css">
     </head>
     <body>
-        <div class="text_block">
-            <div class="text_block-title">
-                <h1>About me</h1>
-            </div>
-            <div class="text_block-info">
-                <p>
-                    Hi! My name is Mariia and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
-                </p>
-                <p>
-                    This page was developed during the course <a href="https://www.mastersacademy.education/frontend-for-beginners-it">'Frontend for beginners' from Masters Academy in 2023.</a>
-                </p>
-                <p>
-                    This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio.
-                </p>
-                <p>
-                    You can contact me via [Links to Socials Nets] and/or check out my GitHub.
-                </p>
-            </div>
+        <div class="wrapper">
+            <header class="header">
+                <div class="container">
+                    <div class="header__inner">
+                        <a class="logo" href="#">
+                            <img src="images/logo.png" alt="icon menu">
+                        </a>
+                        <nav class="menu">
+                            <ul class="menu__list">
+                                <li class="menu__list-item">
+                                    <a class="menu__list-link" href="#">Games</a>
```

Comment:
Hey, @demasha0908 please add a path to file to href, your link doesn't work

Reply:
fixed it